infra/gcp/ensure-prow-build-trusted.sh

```diff
@@ -0,0 +1,47 @@
+#!/usr/bin/env bash
+#
+# Copyright 2020 The Kubernetes Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# This script creates static ip addresses for services deployed in
+# the `prow-build-trusted` cluster
+
+set -o errexit
+set -o nounset
+set -o pipefail
+
+SCRIPT_DIR=$(dirname "${BASH_SOURCE[0]}")
+. "${SCRIPT_DIR}/lib.sh"
+
+PROJECT_NAME="k8s-infra-prow-build-trusted"
+
+function usage() {
+    echo "usage: $0" > /dev/stderr
+    echo > /dev/stderr
+}
+
+if [ $# != 0 ]; then
+    usage
+    exit 1
+fi
+
+ADDRESSES=(
+    "ghproxy-ingress-prod"
+)
+
+for address in "${ADDRESSES[@]}"; do
+    color 6 "Ensure address: $address"
+    ensure_global_address "$PROJECT_NAME" "$address" "IP for ghproxy Ingress"
```

Comment:
How would you feel about trying to do this via terraform instead of via bash?

Alternatively, I guess I'd rather see `ensure-e2e-projects.sh` renamed to something like `ensure-prow-infra.sh` and have this moved into it

Reply:
ok wait actually I don't think we want this at all, we shouldn't be exposing ghproxy

Reply:
I was looking for a way to use google/triage-party running in `aaa` again ghproxy. If ghproxy is not exposed how workloads like triage-party can use it ?

Reply:
Sorry it wasn't clear to me triage party is why you were setting this up.  The recommendation I got from prow folks was to keep ghproxy constrained to within a cluster.  So if your intent is to run ghproxy for triage party, I would run an instance of ghproxy on the `aaa` cluster.

We still need an instance of `ghproxy` on this cluster for the benefit of peribolos jobs etc.

Reply:
Or we can expose it as an "internal" Service on an ILB, which lets a service be used by other members of the VPC?

I don't quite know what is needed here, so guessing a bit.